lib/utils/isOnlyWhitespace.js

```diff
@@ -8,15 +8,12 @@ const isWhitespace = require('./isWhitespace');
  * @param {string} input
  * @returns {boolean}
  */
-module.exports = function (input) {
-	let isOnlyWhitespace = true;
-
+module.exports = function isOnlyWhitespace(input) {
 	for (const element of input) {
 		if (!isWhitespace(element)) {
-			isOnlyWhitespace = false;
-			break;
+			return false;
 		}
 	}
 
-	return isOnlyWhitespace;
+	return true;
```

Comment:
[note] I've simplified the logic to avoid using the same local variable name as the function name.